File: typescript-client-sdk/src/sdk/models/shared/codatpublicapimodelscompanycompanysettings.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";
import { Expose } from "class-transformer";


export class CodatPublicApiModelsCompanyCompanySettings extends SpeakeasyBase {
  @SpeakeasyMetadata()
  @Expose({ name: "companyId" })
  companyId: string;

  @SpeakeasyMetadata()
  @Expose({ name: "offlineConnectorInstall" })
  offlineConnectorInstall: boolean;
}

File: python-client-sdk/src/sdk/models/shared/codat_public_api_models_company_companysettings.py
from __future__ import annotations
import dataclasses
from dataclasses_json import Undefined, dataclass_json
from sdk import utils


@dataclass_json(undefined=Undefined.EXCLUDE)
@dataclasses.dataclass
class CodatPublicAPIModelsCompanyCompanySettings:
    company_id: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('companyId') }})
    offline_connector_install: bool = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('offlineConnectorInstall') }})
    